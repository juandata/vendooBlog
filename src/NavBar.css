
.NavBar {
  width: 152px;
height: 15px;
font-family: Montserrat;
font-weight: 600;
font-size: 12px;
color: #1b153c;
text-decoration: none solid rgb(27, 21, 60);
letter-spacing: 3px;
text-transform: uppercase;
}



