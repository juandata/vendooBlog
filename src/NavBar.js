import React from "react";
import "./NavBar.css";
const NavBar = () => <nav className="NavBar">Latest Updates</nav>;

export default NavBar;
