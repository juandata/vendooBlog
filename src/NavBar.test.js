import React from 'react';
import { shallow } from 'enzyme';
import NavBar from './NavBar';


it('NavBar should render correctly with no props', () => {
    const component = shallow(<NavBar/>);
    
    expect(component).toMatchSnapshot();
  });