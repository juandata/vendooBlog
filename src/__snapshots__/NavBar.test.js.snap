// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar should render correctly with no props 1`] = `
<nav
  className="NavBar"
>
  Latest Updates
</nav>
`;
